torch/_inductor/fx_passes/quantization.py

```diff
@@ -48,12 +48,24 @@
     KeywordArg("w_dtype"),
 )
 
+dequantize_per_channel_to_bf16_weight_pattern = CallFunction(
+    prims.convert_element_type.default,
+    dequantize_per_channel_weight_pattern,
+    KeywordArg("autocast_weight_convert_dtype"),
+)
```

Comment:
maybe use a util function like `dtype_convert(pattern, dtype)` to simplify the places that need a type conversion on the output of the pattern?

Reply:
Thanks for the suggestion. Added `_generate_pattern_with_dtype_convert` as the util function to append pattern with dtype conversion.